### DISCUSSION TITLE
deduplication of search results using asreview data tools

### DISCUSSION THREAD

**Discussion by LilySharon** [asreview/asreview#1210] [Q&A] [answered]
I am PhD student and part of my research is to assess the performance of different automated tools available to expedite Systematic review. I came across your article on ASReview and I have started testing it before using in a properly planned study. 

I ran a search in Scopus and Web of Science, I exported the data and I used ASreview data tools to remove duplicates. ASReview data tools found 45 duplicates in a total of 1883 records. However, when I started the screening process, I realized that there were many articles (in the hundreds) that were duplicates but ASReview data tools didn't find and remove them. 

I Ran the deduplication process in other tools which found hundreds of duplicates such as litsearchr - found more than half of the records as duplicates (could not be accurate), and revtools found between 413-450 depending on the method chosen). My supervisor which ran similar searches and went through the process manually found 171 duplicates.

When I inspected the data these are how some of the ways the titles are rendered "â€œBorders donâ€™t protect areas, people doâ€: insights from the development of an Indigenous Protected and Conserved Area in Kitasoo/Xaiâ€™xais Nation Territory".

How is ASReview review data tools performing the deduplication of records? Could ill-rendering of normal letters into non-letter character or symbols could be issues? How to prevent that to happen?

**Comment by J535D165** [comment, marked as answer]
> How is ASReview review data tools performing the deduplication of records? Could ill-rendering of normal letters into non-letter character or symbols could be issues? How to prevent that to happen?

Yes, this is likely to be the case. ASReview Datatools removes duplicates based on the title and abstract, while other tools might use different metadata (as well). 

What you experience are encoding issues. They can be very annoying and are highly likely caused by your preprocessing. For some users, it might help to avoid the use of Excel. Are you using Excel somewhere in your process?
